### DISCUSSION TITLE
Is there a way to create a macro to move the current file to a specific folder in one go?

### DISCUSSION THREAD

**Discussion by EcovoyCan** [phibr0/obsidian-commander#124] [Q&A]
Is there a way to create a macro to move the current file to a specific folder in one go?

**Comment by MikeKholomeev** [comment]
I did it with [Templater ](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfilemovenew_path-string-file_to_move-tfile) plugin
